src/components/ChatRoom/ChatInput.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { useRecoilState } from 'recoil';
+import { useParams } from 'react-router-dom';
+import styled from '@emotion/styled';
+import useInput from '../hooks/useInput';
+import { Img } from '../styles/UI';
+import { MessageState } from '../../atoms';
+import { OWNER_USER_ID } from '../../constants';
+
+export default function ChatInput() {
+  const { roomId } = useParams<{ roomId: string }>();
+  const [inputValue, handleChange, setInputValue] = useInput<string>('');
+  const [messages, setMessages] = useRecoilState(MessageState);
+
+  const sendMessage = () => {
+    if (inputValue.trim() !== '') {
+      const newMessage = {
+        isOwner: true,
+        userId: OWNER_USER_ID,
+        content: inputValue.trim(),
+        timestamp: new Date().toISOString(),
+      };
+
+      setMessages((prevMessages) =>
+        prevMessages.map((room) => {
+          if (room.roomId === Number(roomId)) {
+            return {
+              ...room,
+              messages: [...room.messages, newMessage],
+            };
+          }
+          return room;
+        })
+      );
+
+      setInputValue('');
+    } else {
+      alert('입력된 메세지가 없습니다.');
+    }
+  };
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    sendMessage();
+  };
+
+  const handleClick = () => {
+    sendMessage();
+  };
+
+  return (
+    <Style.Wrapper>
+      <Style.SearchLogoWrapper>
+        <Img.RoundIcon
+          width="18px"
+          pointer
+          src="/images/search.svg"
+          alt="search"
+        />
+      </Style.SearchLogoWrapper>
+      <form onSubmit={handleFormSubmit}>
+        <Style.TextAreaWrapper
+          type="text"
+          value={inputValue}
+          onChange={handleChange}
+          placeholder="메세지 보내기"
+          autoFocus
+        />
+      </form>
+      <Style.SendMessageLogoWrapper onClick={handleClick}>
+        <Img.RoundIcon
+          width="22px"
+          pointer
+          src="/images/plane.svg"
+          alt="send"
+        />
+      </Style.SendMessageLogoWrapper>
```

Comment:
`SendMessageLogoWrapper`를 `div`가 아닌 `button`으로 구현하고 `type="submit"` 속성을 부여하면 `handleClick`을 별도로 구현하지 않아도 될 것 같아요!

```suggestion
      <form onSubmit={handleFormSubmit}>
        <Style.TextAreaWrapper
          type="text"
          value={inputValue}
          onChange={handleChange}
          placeholder="메세지 보내기"
          autoFocus
        />
        <Style.SendMessageLogoWrapper type="submit">
          <Img.RoundIcon width="22px" pointer src="/images/plane.svg" alt="send" />
        </Style.SendMessageLogoWrapper>
      </form>
```
```javascript
  SendMessageLogoWrapper: styled.button`
    width: 80px;
    height: 36px;
    background-color: #445ffe;
    border-radius: 24px;
    display: flex;
    justify-content: center;
    align-items: center;
  `,
```